bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -770,6 +771,26 @@ public ServerConfiguration setAllowStorageExpansion(boolean val) {
         return this;
     }
 
+    /**
```

Comment:
I am not sure we should put this settings in a bookie config. A server configuration is specific to a bookie instance.

A better solution is:  LocalBookKeeper takes parameter from command line args. e.g. LocalBookKeeper <numBookies> <localbookie config>. so this setting is only for LocalBookKeeper.

Reply:
removed config and moved it to localbookkeeper args